tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "alwaysStrict": true,                     /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-      "noUnusedLocals": true,                /* Report errors on unused locals. */
+      "noUnusedLocals": false,               /* Report errors on unused locals. */
```

Comment:
I thought to make it permanent, because it annoyed me that I can't declare class properties and assign to them, unless i’m using their value somewhere. The case was for `lastResc` and `lastSend` on `Peer`. Let me know if you prefer to keep this rule, and to remove the code that handles these properties, until we’ll actually use them.

Reply:
I figure I can live without it since I still get warnings. Maybe what we can do is be a little more lax on these compiler checks for now, disabling this and similar rules to facilitate development, but then reenable them when the prototype is in a mostly "complete" state where we wouldn't expect unused variables, unimplemented methods, etc...

Reply:
Sounds fine, I think we should start with this one for now, and if other stuff disturbs someone, he can add another.